scripts/histmakers/mw_with_mu_eta_pt.py

```diff
@@ -52,7 +53,15 @@
     if args.unfolding:
         parser = common.set_parser_default(parser, "pt", [32,26.,58.])
 
-    args = parser.parse_args()
+# axes for W MC efficiencies with uT dependence for iso and trigger
+axis_pt_eff_list = [24.,26.,28.,30.,32.,34.,36.,38.,40., 42., 44., 47., 50., 55., 60., 65.]
```

Comment:
Why does this binning and axis need to be defined in the histmaker?

Reply:
Ah ok I see, this was already here, and anyways it's only used for the MC truth efficiencies.